docker/sia/Dockerfile

```diff
@@ -12,5 +12,6 @@ FROM nebulouslabs/sia:1.5.6
 
 COPY --from=sia-builder /go/bin/ /usr/bin/
 
+# rename binaries to old siac siad naming
```

Comment:
Let's remove this from the PR once this is ready to be merged

```suggestion
```

Reply:
These were added to prove that the linters are properly triggered.